thirdeye/thirdeye-frontend/app/pods/components/dimension-heatmap/component.js

```diff
@@ -130,7 +130,7 @@ export default Ember.Component.extend({
       });
       var cell = div.selectAll("g").data(nodes).enter().append("svg:g").attr("class", "cell").attr("transform", function(d) {
         return "translate(" + d.x + "," + d.y + ")";
-      }).on("click", function(d) {
+      // }).on("click", function(d) {
```

Comment:
What's going on here? intentional?

Reply:
this is silly really. The linter was complaining of the unused 'd' variable so I just commented it out. I'm keeping the code here since we will implement an onclick event soon.

Reply:
Ok - got it. All good.